.devcontainer/Dockerfile

```
@@ -1 +1,107 @@
+      libxcb-render-util0 \
+      libxcb-xinerama0 \
+      libxcb-xkb1 \
+    && rm -rf /var/lib/apt/lists/*
+
+# TODO(clairbee): install Miniforge securely
+# TODO(clairbee): install nvm securely
+# TODO: @alexanderilyin: error libmamba Could not open lockfile '/home/vscode/miniforge3/pkgs/cache/cache.lock' \
+# TODO: @alexanderilyin: Run `mamba init`
+# hadolint ignore=DL3004,DL4006,SC1091
+
+RUN \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)